haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/GhcApiSupport.hs

```diff
@@ -187,8 +189,10 @@ withTypechecked globalLock GhcArgs{..} analysisOpts xrefSink
         usedDflags <- getSessionDynFlags
         let env = GhcEnv (showSDoc usedDflags . ppr)
                          (showSDocForUser usedDflags neverQualify . ppr)
-        let extractXref = analyseTypechecked env analysisOpts
-        mapM (parseModule >=> typecheckModule >=> extractXref) (mgModSummaries graph)
+            extractXref = analyseTypechecked env analysisOpts
+            modsum = mgModSummaries graph
```

Comment:
doesn't seem used?

Reply:
Thanks, Robin, for the quick review. Fixed.